presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/SpillableHashAggregationBuilder.java

```diff
@@ -221,7 +227,7 @@ public void close()
 
         hashAggregationBuilder.setOutputPartial();
 
-        Iterator<Page> mergedSpilledPages = MergeHashSort.merge(
+        Iterator<Page> mergedSpilledPages = mergeHashSort.get().merge(
```

Comment:
Add checkState(mergeHashSort.isPresent()) if you think it should always be set at this point. Otherwise add handling of this case.